## 96% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 2
Expected to fail tests: 0
Passed tests: 49
```
```
IDs of failed tests: 702786,703503 
```
<h3>Failed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=seanstrom/beta-tags-chat-communites&PR_ID=21018&APK_NAME=21018.apk&TR_CASE_IDS=702786,703503">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639422">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div># STEP: Check Status community screen</div><div>Device 1: Find `BaseElement` by `xpath`: `//*[@content-desc='community-card-item'][descendant::*[@content-desc='chat-name-text'][@text='Status']]` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:287: in test_community_discovery
    card.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: BaseElement by xpath: `//*[@content-desc='community-card-item'][descendant::*[@content-desc='chat-name-text'][@text='Status']]</b>` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fbfedff31bfe484cb7219beab07a068f?auth=529013089c928eee9cdbee0a2e090435#353">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639390">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div>Device 2: Getting PN by 'user_2' </div><div>Device 2: Looking for a message by text: user_2 </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:958: in test_community_mentions_push_notification
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Push notification with the mention was not received by admin
E    Edited message is not shown correctly for the (receiver) admin
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9043da67bdd04dc583c630aed1f0adb5?auth=c10b37919eeb4ad996df2b44be2c8215#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56af93bbec8e4c6bb8b4d29aefa1a9f2?auth=2f16fa705031ab1fd2fe008bc41a0f4b#113">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (49)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639409">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b044e8d7eb1a4d679af2dfdc9a945314?auth=11a058a9fdcc0a5f7c4ab542c64636a7#127">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cf2d5f49211e4a9f94fb218c1c67a7b3?auth=a6b56123db17470872ad864c32bbca48#97">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639427">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639426">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639401">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9043da67bdd04dc583c630aed1f0adb5?auth=c10b37919eeb4ad996df2b44be2c8215#315">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56af93bbec8e4c6bb8b4d29aefa1a9f2?auth=2f16fa705031ab1fd2fe008bc41a0f4b#271">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639393">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9043da67bdd04dc583c630aed1f0adb5?auth=c10b37919eeb4ad996df2b44be2c8215#196">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56af93bbec8e4c6bb8b4d29aefa1a9f2?auth=2f16fa705031ab1fd2fe008bc41a0f4b#169">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639410">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9043da67bdd04dc583c630aed1f0adb5?auth=c10b37919eeb4ad996df2b44be2c8215#318">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56af93bbec8e4c6bb8b4d29aefa1a9f2?auth=2f16fa705031ab1fd2fe008bc41a0f4b#293">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639423">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9043da67bdd04dc583c630aed1f0adb5?auth=c10b37919eeb4ad996df2b44be2c8215#511">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56af93bbec8e4c6bb8b4d29aefa1a9f2?auth=2f16fa705031ab1fd2fe008bc41a0f4b#472">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639397">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#308">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#282">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639408">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#427">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#392">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639413">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#554">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#464">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639400">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#332">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#301">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639398">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#397">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#362">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639396">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#204">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#134">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639395">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#162">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639425">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#589">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#490">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639406">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#260">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#226">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639399">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#180">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#122">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639415">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ad6ca80d77ce4fe4a99fea3f81e81c7c?auth=5fc9618abdf6dda52dcb28dfad067032#235">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/61286c13ae8745a489c3e78379dad927?auth=f2a8daa85d6aa528d68d040b5216f766#163">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639428">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c96ec65dda994bb79082733e6cfec8d4?auth=c65e5ed75440136cca688a183701e5c5#37">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639405">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/25edecb640204cfeacf1995d40f24e6d?auth=162a674d7be81e75670f8c0f77d23389#462">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fc33151dea7c4a0daf4a1f7a6010980b?auth=aab15db3fa4fffb82dee313c78f56843#294">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639379">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/25edecb640204cfeacf1995d40f24e6d?auth=162a674d7be81e75670f8c0f77d23389#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fc33151dea7c4a0daf4a1f7a6010980b?auth=aab15db3fa4fffb82dee313c78f56843#88">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639384">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/25edecb640204cfeacf1995d40f24e6d?auth=162a674d7be81e75670f8c0f77d23389#355">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fc33151dea7c4a0daf4a1f7a6010980b?auth=aab15db3fa4fffb82dee313c78f56843#213">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639380">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/25edecb640204cfeacf1995d40f24e6d?auth=162a674d7be81e75670f8c0f77d23389#217">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fc33151dea7c4a0daf4a1f7a6010980b?auth=aab15db3fa4fffb82dee313c78f56843#168">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639382">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/25edecb640204cfeacf1995d40f24e6d?auth=162a674d7be81e75670f8c0f77d23389#554">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fc33151dea7c4a0daf4a1f7a6010980b?auth=aab15db3fa4fffb82dee313c78f56843#400">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639394">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/25edecb640204cfeacf1995d40f24e6d?auth=162a674d7be81e75670f8c0f77d23389#413">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fc33151dea7c4a0daf4a1f7a6010980b?auth=aab15db3fa4fffb82dee313c78f56843#263">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639387">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/25edecb640204cfeacf1995d40f24e6d?auth=162a674d7be81e75670f8c0f77d23389#151">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fc33151dea7c4a0daf4a1f7a6010980b?auth=aab15db3fa4fffb82dee313c78f56843#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639419">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/25edecb640204cfeacf1995d40f24e6d?auth=162a674d7be81e75670f8c0f77d23389#493">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fc33151dea7c4a0daf4a1f7a6010980b?auth=aab15db3fa4fffb82dee313c78f56843#340">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639416">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ba01420c1cb64d7291009239ed44cb06?auth=947af51cf588197390e4631a4486d78b#177">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92c265f912b84c3db7dc850a4290e073?auth=2f7e8161dc5733a6c2739ff36ba4bf66#110">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3f8f1082e2f7419c860aed88f211106a?auth=f9487e05d9e5e454f85aea1323e67b40#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639391">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ba01420c1cb64d7291009239ed44cb06?auth=947af51cf588197390e4631a4486d78b#160">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92c265f912b84c3db7dc850a4290e073?auth=2f7e8161dc5733a6c2739ff36ba4bf66#82">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3f8f1082e2f7419c860aed88f211106a?auth=f9487e05d9e5e454f85aea1323e67b40#82">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639392">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ba01420c1cb64d7291009239ed44cb06?auth=947af51cf588197390e4631a4486d78b#324">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92c265f912b84c3db7dc850a4290e073?auth=2f7e8161dc5733a6c2739ff36ba4bf66#207">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3f8f1082e2f7419c860aed88f211106a?auth=f9487e05d9e5e454f85aea1323e67b40#205">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639381">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ba01420c1cb64d7291009239ed44cb06?auth=947af51cf588197390e4631a4486d78b#357">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92c265f912b84c3db7dc850a4290e073?auth=2f7e8161dc5733a6c2739ff36ba4bf66#234">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3f8f1082e2f7419c860aed88f211106a?auth=f9487e05d9e5e454f85aea1323e67b40#232">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639417">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ba01420c1cb64d7291009239ed44cb06?auth=947af51cf588197390e4631a4486d78b#284">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92c265f912b84c3db7dc850a4290e073?auth=2f7e8161dc5733a6c2739ff36ba4bf66#172">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3f8f1082e2f7419c860aed88f211106a?auth=f9487e05d9e5e454f85aea1323e67b40#145">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639420">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ba01420c1cb64d7291009239ed44cb06?auth=947af51cf588197390e4631a4486d78b#477">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/92c265f912b84c3db7dc850a4290e073?auth=2f7e8161dc5733a6c2739ff36ba4bf66#277">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3f8f1082e2f7419c860aed88f211106a?auth=f9487e05d9e5e454f85aea1323e67b40#246">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639421">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f74a4c5c8ec8415ead6f5afa01f37cd8?auth=e7293426f8a5649b35718522c9f08e91#124">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/017e6d489ba04f82b0e0695b963d4594?auth=24effd12f06e9515447b9076db0ddea6#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639388">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f74a4c5c8ec8415ead6f5afa01f37cd8?auth=e7293426f8a5649b35718522c9f08e91#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/017e6d489ba04f82b0e0695b963d4594?auth=24effd12f06e9515447b9076db0ddea6#88">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639389">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f74a4c5c8ec8415ead6f5afa01f37cd8?auth=e7293426f8a5649b35718522c9f08e91#188">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/017e6d489ba04f82b0e0695b963d4594?auth=24effd12f06e9515447b9076db0ddea6#129">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639404">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7a1cd40289b64874bbd4b7f229d1469e?auth=697c470e45d64fc88cef8f1ca6bab3f7#103">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5b85d02318942feb13972b9a361d000?auth=2179dab882e57b0418fae9a8f61edfdc#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639403">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7a1cd40289b64874bbd4b7f229d1469e?auth=697c470e45d64fc88cef8f1ca6bab3f7#60">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5b85d02318942feb13972b9a361d000?auth=2179dab882e57b0418fae9a8f61edfdc#54">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639386">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7a1cd40289b64874bbd4b7f229d1469e?auth=697c470e45d64fc88cef8f1ca6bab3f7#138">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5b85d02318942feb13972b9a361d000?auth=2179dab882e57b0418fae9a8f61edfdc#204">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639383">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fbfedff31bfe484cb7219beab07a068f?auth=529013089c928eee9cdbee0a2e090435#75">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639402">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fbfedff31bfe484cb7219beab07a068f?auth=529013089c928eee9cdbee0a2e090435#51">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639414">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fbfedff31bfe484cb7219beab07a068f?auth=529013089c928eee9cdbee0a2e090435#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639407">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fbfedff31bfe484cb7219beab07a068f?auth=529013089c928eee9cdbee0a2e090435#104">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639418">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fbfedff31bfe484cb7219beab07a068f?auth=529013089c928eee9cdbee0a2e090435#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639424">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/18d800da6100448982349f038b87c7de?auth=024962de8f540ed9e1992a1fba2e916d#59">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639385">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/18d800da6100448982349f038b87c7de?auth=024962de8f540ed9e1992a1fba2e916d#122">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639429">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/18d800da6100448982349f038b87c7de?auth=024962de8f540ed9e1992a1fba2e916d#192">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639412">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c9058dd8f099456a85a565484c8bbb16?auth=9b190af2af93c25cf3c8596fc40142c3#168">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2511506fbd2a419780513589ec85babf?auth=81c67c80586f581d2816fb09fd9b4c20#115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3639411">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c9058dd8f099456a85a565484c8bbb16?auth=9b190af2af93c25cf3c8596fc40142c3#127">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2511506fbd2a419780513589ec85babf?auth=81c67c80586f581d2816fb09fd9b4c20#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/common/home/title_column/view.cljs
+++ b/src/status_im/common/home/title_column/view.cljs
@@ -3,14 +3,30 @@
     [quo.core :as quo]
     [react-native.core :as rn]
     [status-im.common.home.title-column.style :as style]
-    [status-im.common.plus-button.view :as plus-button]))
+    [status-im.common.plus-button.view :as plus-button]
+    [utils.i18n :as i18n]))
 
 (defn view
-  [{:keys [label handler accessibility-label customization-color]}]
+  [{:keys [beta? label handler accessibility-label customization-color]}]
   [rn/view style/title-column
-   [rn/view {:flex 1}
+   [rn/view
+    {:style {:flex           1
+             :align-items    :center
+             :flex-direction :row}}
     [quo/text style/title-column-text
-     label]]
+     label]
+    (when beta?
+      [rn/view
+       {:style {:padding-top    6
+                :padding-bottom 2}}
+       [quo/tag
+        {:accessibility-label :communities-chat-beta-tag
+         :size                32
+         :type                :label
+         :label               (i18n/label :t/beta)
+         :labelled?           true
+         :blurred?            false}]])]
+
    (when handler
      [plus-button/plus-button
       {:on-press            handler

--- a/src/status_im/common/home/title_column/view.cljs
+++ b/src/status_im/common/home/title_column/view.cljs
@@ -21,7 +21,7 @@
                 :padding-bottom 2}}
        [quo/tag
         {:accessibility-label :communities-chat-beta-tag
-         :size                32
+         :size                24
          :type                :label
          :label               (i18n/label :t/beta)
          :labelled?           true

--- a/src/status_im/contexts/chat/home/view.cljs
+++ b/src/status_im/contexts/chat/home/view.cljs
@@ -118,7 +118,8 @@
 (defn- banner-data
   [profile-link]
   {:title-props
-   {:label               (i18n/label :t/messages)
+   {:beta?               true
+    :label               (i18n/label :t/messages)
     :handler             #(rf/dispatch
                            [:show-bottom-sheet {:content chat.actions.view/new-chat}])
     :accessibility-label :new-chat-button}

--- a/src/status_im/contexts/communities/home/view.cljs
+++ b/src/status_im/contexts/communities/home/view.cljs
@@ -64,7 +64,8 @@
 
 (def ^:private banner-data
   {:title-props
-   {:label               (i18n/label :t/communities)
+   {:beta?               true
+    :label               (i18n/label :t/communities)
     :handler             (when config/fast-create-community-enabled?
                            #(rf/dispatch [:show-bottom-sheet {:content actions.home-plus/view}]))
     :accessibility-label :new-communities-button}

--- a/translations/en.json
+++ b/translations/en.json
@@ -186,6 +186,7 @@
   "balance": "Balance",
   "begin-set-up": "Begin setup",
   "below-base-fee": "max fee below base fee",
+  "beta": "Beta",
   "bio": "Bio",
   "bio-added": "Bio added",
   "bio-is-too-long": "Bio is too long",

